Cache_Memcache.php

```diff
@@ -176,9 +176,7 @@ function hard_delete( $key, $group = '' ) {
 	 * @return boolean
 	 */
 	function flush( $group = '' ) {
-		$this->_get_key_version( $group );   // initialize $this->_key_version
-		$this->_key_version[$group]++;
-		$this->_set_key_version( $this->_key_version[$group], $group );
+        $this->_increment_key_version($group);
```

Comment:
As per WordPress code standards, please format using tabs to indent and spaces around parameters.
`		$this->_increment_key_version( $group );`